packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
```

Comment:
Add index.ts and rexport the Code. 

And also reexport on Typography/index.ts

Reply:
Was planning to do that at the end in followup PR. So that this PR can be merged after review. Added the todo for this in description